roles/tftp-server/templates/preseed.cfg.j2

```diff
@@ -331,7 +331,7 @@ d-i apt-setup/local0/key string file:///hd-media/ac583520.asc
 tasksel tasksel/first multiselect standard
 
 # Individual additional packages to install
-d-i pkgsel/include string openssh-server curl
+d-i pkgsel/include string openssh-server curl python python3
```

Comment:
I think you are saying I should do:

d-i pkgsel/include string openssh-server curl ansible  

but ansible doesn't need to be installed on it's target 

personally, I don't mind installing everything everywhere ;)